### PR TITLE
adding new bullshit

### PR DESCRIPTION
**Pull request by rdodev**
Added the following bullshit:
- At scale
- As a service
- Nextify
- Storify
- Personal Brand

### CODE REVIEW

File: SansBullshitSans.ttx [hunk 1]
```diff
@@ -24365,12 +24368,14 @@
               <Ligature components="u,s,h,space,t,h,e,space,e,n,v,e,l,o,p,e" glyph="uniE600"/>
               <Ligature components="e,e,l,i,n,g,space,t,h,e,space,o,n,i,o,n" glyph="uniE600"/>
               <Ligature components="a,t,e,n,t,space,p,e,n,d,i,n,g,space,d,e,s,i,g,n" glyph="uniE600"/>
+              <Ligature components="e,r,s,o,a,l,space,b,r,a,n,d" glyph="uniE600"/>
```
Comment (RoelN) [inline SansBullshitSans.ttx]: Missing an 'n' here :)

Comment (rdodev) [inline SansBullshitSans.ttx]: Ha! Will fix soon. :)

On Wednesday, March 4, 2015, Roel N notifications@github.com wrote:

> In SansBullshitSans.ttx
> https://github.com/RoelN/SansBullshitSans/pull/14#discussion_r25758204:
> 
> > @@ -24365,12 +24368,14 @@
> >                <Ligature components="u,s,h,space,t,h,e,space,e,n,v,e,l,o,p,e" glyph="uniE600"/>
> >                <Ligature components="e,e,l,i,n,g,space,t,h,e,space,o,n,i,o,n" glyph="uniE600"/>
> >                <Ligature components="a,t,e,n,t,space,p,e,n,d,i,n,g,space,d,e,s,i,g,n" glyph="uniE600"/>
> > -              <Ligature components="e,r,s,o,a,l,space,b,r,a,n,d" glyph="uniE600"/>
> 
> Missing an 'n' here :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/RoelN/SansBullshitSans/pull/14/files#r25758204.

## 

Sent from Gmail Mobile

**Comment by rdodev** [comment]
fixed :)

**Comment by RoelN** [comment]
Cheers!

**Pull request merged**